src/tests/SharpRaven.UnitTests/Data/ExceptionFrameTests.cs

```diff
@@ -77,19 +78,49 @@ public void Constructor_InAppFrames_Identified()
                 // The line above will throw.
                 Assert.Fail();
             }
-            catch (Exception exception) {
+            catch (Exception exception)
+            {
                 var frames = new StackTrace(exception, true).GetFrames();
                 var sentryFrames = frames.Select(f => new ExceptionFrame(f)).ToList();
 
                 // The first frame is InApp - (this function).
-                Assert.True(sentryFrames.First().InApp); 
+                Assert.True(sentryFrames.First().InApp);
 
                 // The last frame is InApp (our lambda that divides by zero).
-                Assert.True(sentryFrames.Last().InApp); 
+                Assert.True(sentryFrames.Last().InApp);
```

Comment:
Try to avoid touching code that isn't a part of the problem this PR is fixing, please. 😃